packages/utils/telemetry-utils/src/errorLogging.ts

```diff
@@ -49,56 +54,122 @@ export function extractLogSafeErrorProperties(error: any) {
     return safeProps;
 }
 
-/** Type guard for ILoggingError */
-export const isILoggingError = (x: any): x is ILoggingError => typeof x?.getTelemetryProperties === "function";
+/** Copy props from source onto target, overwriting any keys that are already set on target */
+function copyProps(target: unknown, source: ITelemetryProperties) {
+    Object.assign(target, source);
+}
 
-/**
- * Read-Write Logging Error.  Not exported.
- * This type alias includes addTelemetryProperties, and applies to objects even if not instanceof LoggingError\
- */
-type RwLoggingError = LoggingError;
+/** Metadata to annotate an error object when annotating or normalizing it */
+export interface FluidErrorAnnotations {
+    /** Telemetry props to log with the error */
+    props?: ITelemetryProperties;
+    /** fluidErrorCode to mention if error isn't already an IFluidErrorBase */
+    errorCodeIfNone?: string;
+}
+
+/** Simplest possible implementation of IFluidErrorBase */
+export class SimpleFluidError implements IFluidErrorBase {
+    private readonly telemetryProps: ITelemetryProperties = {};
+
+    readonly errorType: string;
+    readonly fluidErrorCode: string;
+    readonly message: string;
+    readonly stack?: string;
+    readonly name?: string;
+
+    constructor(errorProps: Omit<Omit<IFluidErrorBase, "getTelemetryProperties">, "addTelemetryProperties">) {
```

Comment:
>Omit<Omit<IFluidErrorBase, "getTelemetryProperties">, "addTelemetryProperties"> [](http://example.com/codeflow?start=28&length=79)

Omit<IFluidErrorBase, "getTelemetryProperties" | "addTelemetryProperties">

Reply:
Ah, of course. thanks.